configure.ac

```diff
@@ -226,6 +243,10 @@ else
   export cmake_args
 fi  
 
+case $pyver in
+  3.*) cmake_args="${cmake_args} -DPYTHON_DESIRED:str=3";;
```

Comment:
Should we just add the `2.*` case as well for completeness?

Reply:
Honestly I'd rather not touch this patch again, I tested it in both situations and it worked and I'd rather not re-validate.

In any case it'll need adjusting after we rebase on https://github.com/rpm-software-management/libdnf/pull/497

Reply:
Ahh OK, so this works because our libdnf defaults to py2 right now. Though looking at https://github.com/rpm-software-management/libdnf/pull/433, I see:

```diff
 if (NOT PYTHON_DESIRED)
-    set (PYTHON_DESIRED "2")
+    set (PYTHON_DESIRED "3")
 endif()
```

So this will break once we rebase libdnf.
How about something like:

```bash
case $pyver in
  2.*) cmake_args="${cmake_args} -DPYTHON_DESIRED:str=2";;
  3.*) cmake_args="${cmake_args} -DPYTHON_DESIRED:str=3";;
  *) AC_MSG_ERROR([unexpected \$pyver $pyver]);;
esac
```

and we let our CI validate it?

Reply:
OK, fair enough, done :arrow_down: